pygmt/tests/test_filter1d.py

```diff
@@ -16,7 +16,7 @@
 @pytest.fixture(scope="module", name="data")
 def fixture_table():
     """
-    Load the grid data from the sample earth_relief file.
+    Load the @MaunaLoa_CO2.txt dataset as a pandas dataframe.
     """
     fname = which("@MaunaLoa_CO2.txt", download="c")
     data = pd.read_csv(
```

Comment:
We need to rewrite this function after https://github.com/GenericMappingTools/pygmt/pull/1961 is merged. So let's wait #1961 first.

Reply:
#1961 has been merged. I think it's time to refactor the test using the sample dataset instead.